web/app.fluidity.money/app/util/client-connections.ts

```diff
@@ -1,26 +1,39 @@
 import { PipedTransaction } from "drivers/types";
-import io from "socket.io-client";
 
 const DSSocketManager = ({
   onCallback = (payload: PipedTransaction) => payload,
+  network = ""
 }) => {
-  const socket = io("wss://fanfare.fluidity.money", {
-    transports: ["websocket"],
+  if (network == "") throw new Error(`websocket network to filter for is empty!`);
+
+  const url = new URL("ws://localhost:8888");
+
+  url.pathname = `/${network}`;
+
+  const socket = new WebSocket(url.toString());
+
+  socket.addEventListener("error", (err) => {
+    try {
+      socket.close(); // hygiene?
+    } catch {}
+
+    console.error(`websocket disconnected with err: ${err}`);
```

Comment:
not sure how these event listeners work but should probably at least log the caught error